etc/building-locally.md

```diff
@@ -0,0 +1,241 @@
+# Running Orders Service Locally
+
+## Table of Contents
+
+* [Building the app](#building-the-app)
+* [Setting up MySQL](#setting-up-mysql)
+* [Setting up RabbitMQ](#setting-up-rabbitmq) 
+* [Setting up Zipkin](#setting-up-zipkin) (Optional)
+* [Running the app and stopping it](#running-the-app-and-stopping-it)
+
+### Building the app
+
+To build the application, we used Maven. Maven is a project management tool that is based on the Project Object Model (POM). 
+Typically, people use Maven for project builds, dependencies, and documentation. Maven simplifies the project build. 
+In this task, you use Maven to build the project.
+
+1. Clone this repository and navigate into it.
+
+   ```
+   git clone https://github.com/ibm-cloud-architecture/refarch-cloudnative-micro-orders.git
+   cd refarch-cloudnative-micro-orders/
+   ```
+
+2. Checkout the MicroProfile branch.
+
+   ```
+   git checkout microprofile
+   ```
+
```

Comment:
You need to configure server.xml before this step to set the locations for keystore and truststore